ONSdigital/census-rm-case-processor
# Fix date output format

# Motivation and Context
dates saved in event table event_payload field are in incorrect format


# What has changed
Change DTO date field type to String





# How to test?
NOTE : [Blocked by](https://trello.com/c/29ZsK7ci/825-rmc-197-handle-refusal-received-event-rev13-8)
checkout this branch, build docker image including other on blocked ticket above and run with docker-dev.
run tests and check event table event_payload field for date format




# Links
[Trello](https://trello.com/c/eF4c7Q7e/979-dates-in-case-event-payload-json-not-being-stored-correctly-3)




# Screenshots (if appropriate):
